### DISCUSSION TITLE
egs brachy geometry error

### DISCUSSION THREAD

**Discussion by mehran1234** [clrp-code/egs_brachy#28] [Q&A] [answered]
Hello,
I am using [COMS16mm-I125-6711-hetero.egsinp](https://github.com/clrp-code/egs_brachy/blob/master/egs_brachy/lib/geometry/eye_plaques/sample%20eye%20plaque%20input%20template%20%20files/COMS16mm-I125-6711-hetero.egsinp) input file provided by egs brachy. It gives me a following error after running it "

"Particle started in region 1301 (water_phantom reg 1300) which is outside a source geometry. Please check your geometry"

How can I resolve this error? I am fairly new to egs brachy, any help is greatly appreciated.

regards,
Mehran

**Comment by mchamberland** [comment]
Hmm... let me look into it and get back to you.

**Reply by mehran1234** [reply]
Thank you

**Comment by mchamberland** [comment, marked as answer]
I cannot reproduce this on my end. Typically, this happens if your source of particles extends beyond the boundaries of your source geometry. If you're using the input file provided (which does not work out of the box, sorry about that) as is, then one possibility is that the region discovery process missed some of the voxels of the source. I would recommend you use a higher density of points.
